packages/teleport/src/Discover/Shared.tsx

```diff
@@ -33,13 +33,26 @@ export const Header: React.FC = ({ children }) => (
   </Text>
 );
 
-export const ActionButtons = ({ onProceed }: { onProceed(): void }) => {
+export const ActionButtons = ({
+  onProceed,
+  isProceedDisabled,
+}: {
+  onProceed?(): void;
+  isProceedDisabled?: boolean;
```

Comment:
this name hints that it's a function call when it's' an argument so it should probably be something like 
```suggestion
  disableProceed?: boolean;
```